docs/install-1.7.md

```diff
@@ -1,17 +1,28 @@
+# Installing Service Catalog on Clusters Running Kubernetes 1.7 and Above
```

Comment:
Overall this feels more like the api-aggregation doc than a 1.7 install doc. Maybe we can just have the final "helm command" and link to a separate place where the cert stuff is described.

Reply:
script run + helm command seems simplest to me.

Reply:
done